staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -342,7 +342,7 @@ func (d *Helper) evictPods(pods []corev1.Pod, evictionGroupVersion schema.GroupV
 			if err == nil {
 				returnCh <- nil
 			} else {
-				returnCh <- fmt.Errorf("error when waiting for pod %q terminating: %v", pod.Name, err)
+				returnCh <- fmt.Errorf("error when waiting for pod %q in namespace %q to terminate: %v", pod.Name, pod.Namespace, err)
```

Comment:
I am not good at this, and only want to make it brief.
```suggestion
				returnCh <- fmt.Errorf("error when waiting for pod %q/%q terminating: %v", pod.Namespace, pod.Name, err)
```

Reply:
We expect the `{namespace}/{name}` semantics to be implicitly clear without surrounding words?  There are other `*/{name}` conventions around, such as:

```console
$ kubectl get --help | grep /NAME
  kubectl get [(-o|--output=)j...] (TYPE[.VERSION][.GROUP] [NAME | -l label] | TYPE[.VERSION][.GROUP]/NAME ...) [flags] [options]
```

and while most error-message consumers will be able to distinguish `{namespace}/{name}` from `{type}/{name}`, the fact  that folks will see both makes me personally prefer using a separate word in the error message to make it  clear which string is the namespace.

Reply:
https://github.com/kubernetes/kubernetes/blob/fca9f3774f4e14a998fbe6ea9352586c9efb75a8/staging/src/k8s.io/kubectl/pkg/cmd/clusterinfo/clusterinfo_dump.go#L268-L270

https://github.com/kubernetes/kubernetes/blob/fca9f3774f4e14a998fbe6ea9352586c9efb75a8/staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go#L173-L174

https://github.com/kubernetes/kubernetes/blob/fca9f3774f4e14a998fbe6ea9352586c9efb75a8/staging/src/k8s.io/kubectl/pkg/cmd/run/run.go#L412-L413

This style is widely used in the current code base.

And `pod <ns>/<name>`  is a warning message, not a structured output.

In `kubectl get <x>/<y>`: this is not a log but a command, for the input part,  the x has to be type.

Reply:
If this concerns you, I would suggest doing a cleanup on the current code base(before it, I think this should be discussed in sig-cli.)